frontend/src/pages/SignUp/index.jsx

```diff
@@ -78,6 +53,46 @@ const SignUp = () => {
   const infoUpdateFailMessage = selector((state) => state.auth.infoUpdateFailMessage);
   const infoUpdateLoading = selector((state) => state.auth.infoUpdateLoading);
 
+  useEffect(() => {
+    console.log('stepchange');
+    if(formStep===0){
+      console.log('0');
+    } else if(formStep===1){
+      console.log('1');
+    } else if(formStep===2){
+      console.log('2');
+      dispatch(recommendations('tag', setTags));
+      dispatch(recommendations('uni', setUniversities));
+      dispatch(recommendations('dep', setDepartments));
+    }
+
+  },[formStep]);
```

Comment:
need to delete console.logs and dispatch seems not to be using actual redux reducer, instead sending the setters. We should probably refactor that

Reply:
Done.